src/main/java/homework_2/pyramid_printer/PyramidPrinter.java

```diff
@@ -0,0 +1,47 @@
+package homework_2.pyramid_printer;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class PyramidPrinter {
```

Comment:
![image](https://user-images.githubusercontent.com/58872509/131849385-6bc6b598-edeb-4700-a501-886e87ddd3e4.png)

Reply:
![image](https://user-images.githubusercontent.com/58872509/131849451-62b2a595-b645-4c56-ad58-40aaaddfc5d8.png)
Same for -1 and 5.4

Reply:
isValid method fixed